### PR TITLE
Add StreetDetail

### PR DESCRIPTION
**Pull request by Lax125**
Signed-off-by: Marcus Koh <marcus.koh@zepben.com>

### CODE REVIEW

**Comment by ryanjcoleman** [comment]
Task linked: [EWB-1069 Update StreetAddress in Python SDK](https://app.clickup.com/t/6929263/EWB-1069)

**Comment by Lax125** [comment]
Issues possibly arising from new protobuf version. Will look into it.

**Comment by Lax125** [comment]
Having an issue with multiple inheritance here too (deja vu...)
I'll work on it tomorrow.

**Comment by Lax125** [comment]
After stepping away from the code, I'm now considering just merging all the fields from `TableStreetDetails` and `TableTownDetails` into `TableStreetAddresses`. I already did it for `TableStreetDetails` in the jvm sdk and all usages of `TableStreetDetails` and `TableTownDetails` seem to be a part of a more monolithic action on a `TableStreetAddresses`.

**Comment by Lax125** [comment]
I wonder if it would be better to rename the columns to be more specific (e.g. type -> street_type) for both sdks.

**Comment by Lax125** [comment]
Will go over the changes on the jvm version.

**Review by charlta — CHANGES_REQUESTED** [review]
I will push some changes and a test that fails to reveal an issue with PR. Can you please fix so the test passes and also add tests for the `TownDetail` and `StreetDetails` empty checks (as per JVM)

File: test/services/network/translator/test_network_translator.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import pytest
 from hypothesis import given, HealthCheck, settings
 
-from database.sqlite.schema_utils import assume_non_blank_street_address_details
+from test.database.sqlite.schema_utils import assume_non_blank_street_address_details
```
Comment (charlta) [inline test/services/network/translator/test_network_translator.py:9]: Good catch on the import. This is my pet peeve as the import works in the IDE without the `test.` so I constantly miss it.

**Review by charlta — APPROVED** [review]
